From 97788f034d909159b00091a7b4fcc1355d9a8112 Mon Sep 17 00:00:00 2001
From: Lu Yahan <yahan@iscas.ac.cn>
Date: Thu, 13 May 2021 12:56:05 +0800
Subject: [PATCH] [riscv64] Skip incompatible inspector test

Bug: v8:11767

Change-Id: I8ddd578bbc60c72c8d8e269da5492e6fce68482d
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/2894025
Reviewed-by: Jakob Gruber <jgruber@chromium.org>
Commit-Queue: Jakob Gruber <jgruber@chromium.org>
Auto-Submit: Yahan Lu <yahan@iscas.ac.cn>
Cr-Commit-Position: refs/heads/master@{#74586}
---
 test/inspector/inspector.status | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/test/inspector/inspector.status b/test/inspector/inspector.status
index de6be135712..17927b3be61 100644
--- a/test/inspector/inspector.status
+++ b/test/inspector/inspector.status
@@ -129,6 +129,9 @@
   # SIMD support is still in progress.
   'debugger/wasm-scope-info*': [SKIP],
   'debugger/wasm-step-after-trap': [SKIP],
+
+  # https://github.com/riscv/v8/issues/490
+  'cpu-profiler/console-profile-wasm': [SKIP],
 }],  # 'arch == riscv64'
 
 ['arch == riscv64 and variant == stress_incremental_marking', {
-- 
2.35.1

